### PR TITLE
Remove 2.2.0 / 2.3.0 deprecations

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Corresponding help changes: https://github.com/zaproxy/zap-core-help/pull/414

**Comment by psiinon** [comment]
Ugh, I had run spotless apply but hadnt noticed it was failing locally :/

**Comment by psiinon** [comment]
Doh! - trailing comma on a list ... should ok ok now ...

File: zap/src/main/java/org/parosproxy/paros/db/paros/ParosTableHistory.java [hunk 1]
```diff
@@ -44,6 +44,8 @@
 // ZAP: 2019/06/01 Normalise line endings.
 // ZAP: 2019/06/05 Normalise format/style.
 // ZAP: 2020/11/26 Use Log4j 2 classes for logging.
+// ZAP: 2022/02/03 Removed getHistoryList(long, int) and getHistoryList(long)</li>
```
Comment (kingthorin) [inline zap/src/main/java/org/parosproxy/paros/db/paros/ParosTableHistory.java:47]: Drop the li tag?

Comment (psiinon) [reply zap/src/main/java/org/parosproxy/paros/db/paros/ParosTableHistory.java]: Yes- good spot :) Done.

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**